scripts/anvil_tools/gather_and_concatenate_data_model_tsvs.py

```diff
@@ -0,0 +1,81 @@
+"""Gather a single entity's data model metadata by workspace_name (and workspace_project) into a single load tsv file format and report in excel document.
+
+Usage:
+    > python3 gather_and_concatenate_data_model_tsvs.py -x EXCEL_FILE -e ENTITY_TABLE_NAME"""
+
+import argparse
+from firecloud import api as fapi
+from openpyxl import load_workbook
+import pandas as pd
+
+
+def gather_and_concatenate_data_model_tsvs(input_file, entity_name):
+    """Get data table tsv files from list of workspaces and concatenate results into a single excel report."""
+
+    # read full excel sheet into dataframe - all rows of workspace project and workspace names
+    workspace_info = pd.read_excel(input_file, sheet_name="Sheet1", index_col=None)
+
+    # instantiate empty list to hold all entity information from all workspaces
+    all_workspace_entities = []
+    failed_workspaces = []
+    # for each workspace_name, workspace_project pair
+    for index, workspace in workspace_info.iterrows():
```

Comment:
does this break if there are not exactly 2 columns in the input file? is there any check to make sure the input file is formatted correctly with the right order of columns?

Reply:
so order of the columns shouldn't matter since im getting columns with their exact names. I can test it but it also should be able to have additional columns but at minimum, has the two that the script is looking to parse workspace information from.